cwiczenie3/osoba.php

```diff
@@ -13,14 +14,23 @@ function __construct($imie, $nazwisko, $plec, $rok)
         $this->plec = $plec;
         $this->rok = $rok;
     }
-
     public function powitaj()
     {
         return "Czesc mam na imie " . $this->imie;
     }
-
     public function obliczWiek()
     {
         return " i mam " . (date('Y') - ($this->rok)) . " lata";
     }
+
+    public function Imie()
+    {
+      return $this->imie;
+    }
+    public function dodajZnajomego(Osoba $osoba)
+    {
+
+        $this->przyjaciel = $osoba->Imie();
```

Comment:
Dodaj calego przyjaciela a nie tylko imie - dodajac tylko imie nie jestes w stanie wykonac nastepnego cwiczenia: stworz metode porownujaca roznice wieku pomiedzy przyjaciolmi (`porownajWiekZPrzyjacielem()`), ktora bedzie zwracac `Roznica wieku miedzy nami to {LICZBA} lat`.